source/elements/oneMKL/source/domains/rng/onemkl-rng-overview.rst

```diff
@@ -35,42 +35,21 @@ In computational statistics, random variate generation is usually made in two st
       in order to generate (or imitate) random variates and random
       vectors from arbitrary distributions.
 
-.. rubric:: Structure
 
-RNG domain contains two classes types:
+.. rubric:: Execution Models
 
-   -  Engines (basic random number generators) classes, which holds
-      the state of generator and is a source of i.i.d. random. Refer to
-      :ref:`onemkl_rng_engines_basic_random_number_generators`
-      for a detailed description.
-   -  Distribution classes templates (transformation classes) for different types of statistical
-      distributions, for example, uniform, normal (Gaussian), binomial,
-      etc. These classes contain all of the distribution’s parameters
-      (including generation method). Refer to :ref:`onemkl_rng_distributions` for
-      a detailed description of the distributions.
+RNG domain supports two execution models:
 
-The RNG domain also contains two types of free functions:
+   #. :ref:`Host API<onemkl_rng_manual_offload_routines>`, which is aligned with the rest of oneMKL domains 
+      :ref:`onemkl_member_functions`.
```

Comment:
lets change it to :ref:`oneMKL domains<onemkl_member_functions>`

Reply:
done